shell/platform/darwin/macos/framework/Source/FlutterPlatformViewController.h

```diff
@@ -17,12 +17,14 @@
 @interface FlutterPlatformViewController ()
 
 /**
- * Creates a platform view of viewType with viewId.
+ * Creates a platform view of viewType with viewId and arguments passed from
+ * the framework's creationParams constructor parameter.
  * FlutterResult is updated to contain nil for success or to contain
  * a FlutterError if there is an error.
  */
 - (void)onCreateWithViewID:(int64_t)viewId
```

Comment:
Optional opportunistic nit: this should really be `Identifier`, rather than `ID`.

Reply:
Good call. Done!